### PR TITLE
Initial package for sgkit.

### PR DESCRIPTION
**Pull request by jeromekelleher**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jeromekelleher** [comment]
Hooray, looks like this is working!

@eric-czech, would you mind having a quick look over please? Also, can I put you down as a maintainer (to reduce the bus factor)?

I've made it a "noarch" package, which is the right approach I think, since we're not shipping any compiled code.

**Comment by jeromekelleher** [comment]
@eric-czech, @tomwhite any takers for reviewing this and volunteering to be named as a maintainer? There's no much involved, just clicking "merge" on auto update PRs every now and again.

**Review by eric-czech — APPROVED** [review]

**Comment by eric-czech** [comment]
> @eric-czech, would you mind having a quick look over please? Also, can I put you down as a maintainer (to reduce the bus factor)?

Hey @jeromekelleher, yea go for it.  Sorry for the delay.

**Comment by tomwhite** [comment]
@jeromekelleher happy to be added as a maintainer if you need another one.

**Comment by jeromekelleher** [comment]
Thanks @eric-czech and @tomwhite, I've added you both as maintainers.

**Comment by jeromekelleher** [comment]
@conda-forge/staged-recipes this is ready for final review and merge :heart:

File: recipes/sgkit/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    - sgkit
+
+about:
+  home: https://pystatgen.github.io/sgkit
+  license: Apache-2.0
+  license_family: APACHE
+  license_file: LICENSE
+  summary: "Statistical genetics toolkit"
+  doc_url: https://pystatgen.github.io/sgkit
+  dev_url: https://github.com/pystatgen/sgki
```
Comment (dopplershift) [inline recipes/sgkit/meta.yaml:59]:
```suggestion
  dev_url: https://github.com/pystatgen/sgkit
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a minor typo, then this is ready to go in.

File: recipes/sgkit/meta.yaml (hunk 1 above)
Comment (jeromekelleher) [reply recipes/sgkit/meta.yaml]: Doh! :facepalm: Thanks @dopplershift

**Review by dopplershift — APPROVED** [review]

**Comment by jeromekelleher** [comment]
:tada: Thanks @dopplershift !

**Pull request merged**